include/deal.II/lac/sparsity_tools.h

```diff
@@ -103,6 +103,33 @@ namespace SparsityTools
                   const Partitioner partitioner = Partitioner::metis
                  );
 
+  /**
+   * Using a coloring algorithm provided by ZOLTAN to color indices whose
+   * connections are represented using a SparsityPattern object. In effect,
+   * we view this sparsity pattern as a graph of connections between nodes,
+   * where each nonzero entry in the sparsity pattern corresponds to
+   * an edge between two nodes in the connection graph. The goal is to assign
+   * each node a color index such that no two connected nodes have
+   * same color index. Note that this function requires that SparsityPattern be
```

Comment:
`require the @p sparsity_pattern to be symmetric`

Reply:
-> have THE same color index